<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockProdutos</class>
 <widget class="QWidget" name="StockProdutos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>671</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>671</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>671</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>671</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1010</width>
     <height>671</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1010</width>
     <height>671</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ADCE74;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page1_Stock">
    <widget class="QPushButton" name="btnInicio_stock">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>95</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
     </property>
     <property name="text">
      <string> Início</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/img/Home.png</normaloff>:/img/img/Home.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>711</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #004b23;</string>
     </property>
     <property name="text">
      <string>Stock de Produtos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>460</y>
       <width>471</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <fontweight>DemiBold</fontweight>
      </font>
     </property>
     <property name="title">
      <string>Pesquisar por:</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Gama </string>
      </property>
     </widget>
     <widget class="QLabel" name="lblTotalProdutos">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <fontweight>ExtraBold</fontweight>
       </font>
      </property>
      <property name="text">
       <string>Produtos:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPesquisarProduto_stock">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <fontweight>DemiBold</fontweight>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(196, 196, 196);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/img/img/pesquisar.png</normaloff>:/img/img/pesquisar.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Código</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCodigo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#DAF7D9;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtProduto">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#DAF7D9;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Produto</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbPesquisarGama">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#DAF7D9;</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vinhos de Mesa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alimentação</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cervejas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Espumantes e Sidras</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Licores e Aguardentes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Águas e Refrescos</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnLimparPesquisa">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <fontweight>DemiBold</fontweight>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(196, 196, 196);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/img/img/Refresh.png</normaloff>:/img/img/Refresh.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget_stock">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>721</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #E6F7E6;</string>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>70</y>
       <width>201</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>ALTERAR STOCK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="btnSaidaStock">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>121</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
      </property>
      <property name="text">
       <string> Saída Diversa</string>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/img/img/Remove.png</normaloff>:/img/img/Remove.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtQuantidade">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#DAF7D9;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Código do Produto</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCodigoProduto">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#DAF7D9;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEntradaStock">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>121</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Entrada Diversa</string>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/img/img/Add.png</normaloff>:/img/img/Add.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Quantidade</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lblErro">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
